src/reactive-handler.ts

```diff
@@ -107,20 +101,7 @@ export class ReactiveProxyHandler {
         return ArrayConcat.call(getOwnPropertyNames(originalTarget), getOwnPropertySymbols(originalTarget));
     }
     isExtensible(shadowTarget: ReactiveMembraneShadowTarget): boolean {
-        const shadowIsExtensible = isExtensible(shadowTarget);
-
-        if (!shadowIsExtensible) {
-            return shadowIsExtensible;
-        }
-
-        const { originalTarget, membrane } = this;
-        const targetIsExtensible = isExtensible(originalTarget);
-
-        if (!targetIsExtensible) {
-            lockShadowTarget(membrane, shadowTarget, originalTarget);
-        }
-
-        return targetIsExtensible;
+        return isExtensibleMembraneTrap.call(this, shadowTarget);
```

Comment:
Should we abstract the `.call` away?

Reply:
I'm fine with removing that extra call and focus on traps that expects the handler as the first argument. It is ok since we calle them MembraneTrap, and we can say that MembraneTrap is different from proxy traps by passing the handler as the first argument...

Reply:
this is now abstracted out... no more .calls